angular.json

```diff
@@ -64,7 +64,12 @@
             "aot": true,
             "assets": [
               "apps/datafeeder/src/favicon.ico",
-              "apps/datafeeder/src/assets"
+              "apps/datafeeder/src/assets",
+              {
+                "glob": "*",
+                "input": "translations",
+                "output": "assets/i18n/"
+              }
```

Comment:
Sounds great! But the translations are still part of the assets right? https://github.com/geonetwork/geonetwork-ui/tree/6e0ebc5f8e75c008995e921688a7f14c11881d08/apps/datafeeder/src/assets/i18n

Shouldn't they be removed? See https://github.com/geonetwork/geonetwork-ui/pull/170 for inspiration :)